templates/csi.yaml.j2

```diff
@@ -232,7 +232,7 @@ spec:
           args:
             - "--provisioner=kadalu"
             - "--csi-address=$(ADDRESS)"
-            - "--connection-timeout=15s"
+            - "--connection-timeout=2m"
```

Comment:
Should it be few seconds more? I see 'initialDelay' itself is 120 seconds. I think we may have to give some delta (at least of 3-7 seconds) between these 2?

Reply:
Ok, I will do that